### PR TITLE
build: Prepare 0.6.2 release

### PR DESCRIPTION
**Pull request by pameladelgado**

### CODE REVIEW

File: CHANGELOG.rst [hunk 1]
```diff
@@ -6,16 +6,52 @@
...
+
+Notable improvements
+~~~~~~~~~~~~~~~~~~~~
+
+🚄 Datasets: dataset creation and import unified in the UI
+
+Bug fixes
+~~~~~~~~~
+
+* Datasets now show file listing with folder hierarchy
```
Comment (ableuler) [inline CHANGELOG.rst:22]:
```suggestion
* Datasets now include the folder hierarchy in file listings
```

File: CHANGELOG.rst [hunk 2]
```diff
@@ -6,16 +6,52 @@
...
+* Datasets: avoid recursive addition of data directory in Renku CLI
+* Datasets: fixed export to Dataverse
+* Datasets: fix metadata commit after `renku dataset unlink`
+* Environments: improve styling
+* Hiding/showing code cells is now possible from the UI
+
+Miscellaneous
+^^^^^^^^^^^^^
+
+- A maintenance page can now be displayed for when Renkulab is undergoing an scheduled maintenance
```
Comment (ableuler) [inline CHANGELOG.rst:32]:
```suggestion
- A maintenance page can now be displayed for when Renkulab is undergoing a scheduled maintenance
```

File: CHANGELOG.rst [hunk 3]
```diff
@@ -6,16 +6,52 @@
...
+- Help page and dropdown contain links to Renku and Renku CLI Docs
```
Comment (ableuler) [inline CHANGELOG.rst:33]:
```suggestion
- Help page and dropdown contain links to Renku and Renku CLI docs
```

File: CHANGELOG.rst [hunk 4]
```diff
@@ -6,16 +6,52 @@
...
+- Easy UI access to Gitlab, user settings and user profile
```
Comment (ableuler) [inline CHANGELOG.rst:34]:
```suggestion
- Easy UI access to user settings and user profile on Gitlab
```

File: CHANGELOG.rst [hunk 5]
```diff
@@ -6,16 +6,52 @@
...
+Notable improvements
+~~~~~~~~~~~~~~~~~~~~
+
+🚄 Datasets: dataset creation and import unified in the UI
+
+Bug fixes
+~~~~~~~~~
+
+* Datasets now show file listing with folder hierarchy
+* Datasets: avoid recursive addition of data directory in Renku CLI
```
Comment (ableuler) [inline CHANGELOG.rst:23]: I would suggest "of data directories" or "of the data directory". Not sure which version is actually the correct one for this change.

File: CHANGELOG.rst [hunk 6]
```diff
@@ -6,16 +6,52 @@
...
+* Datasets: fixed export to Dataverse
```
Comment (ableuler) [inline CHANGELOG.rst:24]:
```suggestion
* Datasets: fix export to Dataverse
```

**Comment by ableuler** [comment]
I added a few language suggestions. Really looking forward to having this out in the wild 🎉

File: CHANGELOG.rst [hunk 7]
```diff
@@ -6,16 +6,52 @@
...
+- The Tensorboard Renku docker image with Cuda and `Tensorflow 1.14 <https://www.tensorflow.org/>`__ is now available with the latest Renku `0.10.3`. To use it just replace the first line of your Dockerfile with `FROM renku/renkulab:renku0.10.3-cuda10.0-tf1.14-0.6.2`
+- Docker images in project templates use Renku `0.10.3 <https://github.com/SwissDataScienceCenter/renku-python/releases/tag/0.10.3>`__
+
+Individual components
+~~~~~~~~~~~~~~~~~~~~~
+
+For changes to individual components, check:
+
+* renku ui `0.9.2 <https://github.com/SwissDataScienceCenter/renku-ui/releases/tag/0.9.2>`__
+* renku-notebooks `0.7.2 <https://github.com/SwissDataScienceCenter/renku-notebooks/releases/tag/0.7.2>`__
```
Comment (rokroskar) [inline CHANGELOG.rst:46]:
```suggestion
* renku-notebooks `0.7.3 <https://github.com/SwissDataScienceCenter/renku-notebooks/releases/tag/0.7.3>`__
```

File: CHANGELOG.rst [hunk 8]
```diff
@@ -6,16 +6,54 @@
...
+* Environments: improve styling
+
+Miscellaneous
+^^^^^^^^^^^^^
+
+- A maintenance page can now be displayed for when Renkulab is undergoing a scheduled maintenance 🔧
+- Help page and dropdown contain links to Renku and Renku CLI documentation 📖
+- Easy UI access to GitLab projects, user settings and user profile 👤
+- Python environments now include a plugin to monitor memory usage visually 📈
+- A new Renku docker image with `Julia <https://julialang.org/>`__ is now available. 📣 To use it just replace the first line of your Dockerfile with `FROM renku/renkulab:renku0.10.3-julia1.3.1-0.6.2`
```
Comment (rokroskar) [inline CHANGELOG.rst:38]:
```suggestion
- A new Renku docker image with `Julia <https://julialang.org/>`__ is now available. 📣 To use it just replace the first line of your Dockerfile with ``FROM renku/renkulab:renku0.10.3-julia1.3.1-0.6.2``
```

File: CHANGELOG.rst [hunk 9]
```diff
@@ -6,16 +6,54 @@
...
+- The Tensorflow Renku docker image with Cuda and `Tensorflow 1.14 <https://www.tensorflow.org/>`__ is now available with the latest Renku `0.10.3`. To use it just replace the first line of your Dockerfile with `FROM renku/renkulab:renku0.10.3-cuda10.0-tf1.14-0.6.2`
```
Comment (rokroskar) [inline CHANGELOG.rst:39]:
```suggestion
- The Tensorflow Renku docker image with Cuda and `Tensorflow 1.14 <https://www.tensorflow.org/>`__ is now available with the latest Renku ``0.10.3``. To use it just replace the first line of your Dockerfile with ``FROM renku/renkulab:renku0.10.3-cuda10.0-tf1.14-0.6.2``
```

**Pull request merged**